components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/openidconnect/DefaultIDTokenBuilder.java

```diff
@@ -797,4 +803,59 @@ private String getIdpSessionKey(String accessToken) {
         }
         return idpSessionKey;
     }
+
+    /**
+     * Create jwtClaimsSetBuilder for requested essential claims. This method is used in the Authorization Code flow.
+     *
+     * @param jwtClaimsSetBuilder   JWT Claim Set.
```

Comment:
Shall we keep one format for the method comment? ie, if we are using full stops for the param description shall we do the same in other places as well? WDYT?